app/components/containerConsole/containerConsole.html

```diff
@@ -33,8 +33,13 @@
                 </select>
               </div>
             </div>
+            <div class="col-sm-3">
+              <div class="input-group">
+                <input class="form-control" type="text" name="user" ng-model="state.user" placeholder="User">
+              </div>
+            </div>
             <!-- !command-list -->
-            <div class="col-sm-8">
+            <div class="col-sm-5">
```

Comment:
I'm not fond of the current layout.

I don't really like the actual layout for the console panel.

![portainer 11](https://user-images.githubusercontent.com/5485061/27830515-13105878-60c7-11e7-9565-37e66ec006d2.png)

Could you update it to have the command and the user on two lines? And add labels ("Command", "User") in front of each line ?

It'd be nice to have a tooltip next to the user label with the following content:  `The user, and optionally, group to run the exec process inside the container. Format is one of: user, user:group, uid, or uid:gid`.